digit_recognition_trust_owner/mse_src/app.py

```diff
@@ -0,0 +1,60 @@
+"""app module."""
+
+import base64
+import io
+import logging
+import multiprocessing
+import os
+import threading
+from http import HTTPStatus, client
+from typing import Any, Optional
+
+import numpy as np
+from flask import Flask, Response, jsonify, request
+from flask_cors import CORS
+from keras.models import load_model
+from PIL import Image
+
+app = Flask(__name__)
+cors = CORS(app)
+
+logging.basicConfig(
+    format="[%(levelname)s] %(message)s",
+    level=logging.DEBUG
+)
+
+CWD_PATH = os.getenv("MODULE_PATH")
+if (CWD_PATH == None):
+    model = load_model(f"./mnist.h5")
+else:
+    model = load_model(f"{CWD_PATH}/mnist.h5")
```

Comment:
What is the idea behind this `if...else`? Could we just trust in Cosmian to provide `MODULE_PATH` value?

Reply:
ok if you prefer - the idea was to prevent an error if env var missing and this is equal to None (which was the case on my machine locally)

Reply:
Totally agree concerning your local machine. However, if such a behaviour happens on the sgx: it's our fault and we have a bigger issue. 
I think the user needs to trust in us and do not suppose it can fail. The fallback you have implemented is not documented so i would suggest to simplify the code (on a pedagogic way) and just use the value from the env. In the README, you can put a memo: if the user wants to run the example on its local host, the user needs to specify the env variable before running flask. What do you think?

Reply:
fine for me ! 👍